docs/content/concepts/partitions-schedules-sensors/sensors.mdx

```diff
@@ -281,6 +281,21 @@ For more information on resources, refer to the [Resources documentation](/conce
 
 ---
 
+## Logging in sensors
```

Comment:
Is there a reason these got moved up in the file?

Reply:
I moved the section up on purpose. In my mind, it makes more sense to talk about all the features of instigators (sensors/schedules) + what you can include in the evaluation function before moving on to things after you have your instigator definition, such as testing and monitoring.